### ISSUE TITLE
Search never returns more than 20 items.

### ISSUE THREAD

**Issue by Coppyhop** [AurousGroup/Bug-Tracker#62]
Search works great, except I can never get it to load more than 20 items, normally it loads 19, I highly doubt there's only 19 Beatles songs on Aurous.

**Comment by Saeron** [comment]
Yes, looks like a program limitation, but there is not a button for show more.

**Comment by sylvainlap** [comment]
+1
